### PR TITLE
montserrat: 1.0 -> 7.210

### PR DESCRIPTION
**Pull request by 06kellyjac**
###### Motivation for this change

- Bump version to `7.210`
- nixpkgs-fmt

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112527` at 3aae1a7d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/0e0347283cf84d87fd9c885b5e7cdf11">montserrat</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/data/fonts/montserrat/default.nix:14:3:
```
   |
14 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112527` at 3aae1a7d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/f3fdd731c51596d51a326b7cac273ea5">montserrat</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/data/fonts/montserrat/default.nix:14:3:
```
   |
14 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

**Comment by scolobb** [comment]
Thank you @06kellyjac for this PR which seriously improves this package!

I'd say you can squash the two commits, given that the second commit actually changes quite a bit of things in the file beyond the version.

You may also want to add yourself to the maintainers of this package.

**Review by scolobb — CHANGES_REQUESTED** [review]
Squash the two commits and maybe add yourself to the maintainers.

**Comment by 06kellyjac** [comment]
Thanks @scolobb . I'll get on that soon

**Review by scolobb — APPROVED** [review]

File: pkgs/data/fonts/montserrat/default.nix [hunk 1]
```diff
@@ -1,29 +1,28 @@
-# Originally packaged for ArchLinux.
-#
-# https://aur.archlinux.org/packages/ttf-montserrat/
+{ stdenv, lib, fetchFromGitHub }:
 
-{ lib, fetchzip }:
+stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/montserrat/default.nix:3]: fonts normally do not use mkDerivation but just the fetch part and postFetch to move files.

Comment (06kellyjac) [reply pkgs/data/fonts/montserrat/default.nix:3]: Ok ill re-jig it

Comment (06kellyjac) [reply pkgs/data/fonts/montserrat/default.nix]: rejigged

**Comment by 06kellyjac** [comment]
ready for review

**Comment by SuperSandro2000** [comment]
```
these derivations will be built:
  /nix/store/v0a7169ylmswr6b9vgpv47nvivixg9d6-montserrat-7.210.drv
building '/nix/store/v0a7169ylmswr6b9vgpv47nvivixg9d6-montserrat-7.210.drv' on 'ssh://server7'...

trying https://github.com/JulietaUla/montserrat/archive/v7.210.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   129    0   129    0     0    327      0 --:--:-- --:--:-- --:--:--   326
100 15.1M    0 15.1M    0     0   203k      0 --:--:--  0:01:16 --:--:--  202k
install: missing file operand
Try 'install --help' for more information.
error: build of '/nix/store/v0a7169ylmswr6b9vgpv47nvivixg9d6-montserrat-7.210.drv'
```

**Comment by 06kellyjac** [comment]
resolved

**Pull request merged**